LibertyDSNP/example-client
# Feature/login quick start guide/178789849

Purpose
---------------
feature: add how-to guide to setting up metamask and torus


Solution
---------------
add instructions text and styling


Change summary
---------------
* added text
* met with heather to discuss styling
* implemented styling
* added conditions for if metamask is setup or not


Steps to Verify
----------------
1. run code
2. make sure that the copy is descriptive enough.
3. try in different browsers
4. try in browser with and without metamask installed

Additional details / screenshot
----------------
- Any suplmenental pictures or material
- ![Screenshot]()
